### PR TITLE
chore: add html to prettier for packages that include html files

### PR DESCRIPTION
**Pull request by janechu**
# Description

This adds prettier formatting to packages that include HTML files.

## Motivation & context

This change was sparked by https://github.com/microsoft/fast/pull/3492 which shows prettier changes that may be set globally if html files are not specified.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components-msft/src/text-area/fixtures/text-area.html [hunk 1]
```diff
@@ -61,12 +61,20 @@ <h5>Disabled</h5>
...
+        disabled
+        placeholder="placeholder"
+    ></fast-text-area>
 
     <!-- Read only -->
     <h5>Read only</h5>
-    <fast-text-area appearance="filled" value="Readonly text area" readonly>label</fast-text-area>
-    <fast-text-area appearance="filled" value="Readonly text area" readonly>label</fast-text-area>
+    <fast-text-area appearance="filled" value="Readonly text area" readonly
+        >label</fast-text-area
```
Comment (nicholasrice) [inline packages/web-components/fast-components-msft/src/text-area/fixtures/text-area.html:73]: o.0

Comment (janechu) [reply packages/web-components/fast-components-msft/src/text-area/fixtures/text-area.html:73]: Oh, that is interesting... i'll see if i can change some settings

File: packages/web-components/fast-components/src/checkbox/fixtures/base.html [hunk 2]
```diff
@@ -16,12 +16,16 @@ <h4>Indeterminate</h4>
...
 
     <!-- Disabled -->
-    <h4>Disabled</h1>
+    <h4>Disabled</h4>
     <fast-checkbox disabled></fast-checkbox>
     <fast-checkbox disabled>label</fast-checkbox>
     <fast-checkbox disabled checked>checked</fast-checkbox>
-    <fast-checkbox disabled checked class="flag-indeterminate">Indeterminate checked</fast-checkbox>
-    <fast-checkbox disabled class="flag-indeterminate">Indeterminate unchecked</fast-checkbox>
+    <fast-checkbox disabled checked class="flag-indeterminate"
```
Comment (nicholasrice) [inline packages/web-components/fast-components/src/checkbox/fixtures/base.html:23]: I wonder why it's getting confused here

File: packages/web-components/fast-components/src/radio-group/fixtures/base.html [hunk 3]
```diff
@@ -4,36 +4,46 @@ <h1>Radio Group</h1>
     <h4>Defaults</h4>
     <div>
         <fast-radio-group name="numbers">
-            <label style="color: ${neutralForegroundRestBehavior.var};" slot="label">Numbers</label>
+            <label style="color: ${neutralForegroundRestBehavior.var};" slot="label"
```
Comment (nicholasrice) [inline packages/web-components/fast-components/src/radio-group/fixtures/base.html:7]: Not yours, but I bet this interpolation doesn't work

Comment (janechu) [reply packages/web-components/fast-components/src/radio-group/fixtures/base.html:7]: Hmmm, no I wouldn't think so

File: sites/fast-tooling-examples/app/examples/native-element-1/index.html [hunk 4]
```diff
@@ -1,33 +1,21 @@
...
-</body>
-
-</html>
\ No newline at end of file
+    <body>
+        <div id="root">
+            <input id="text" />
+            <pre id="data-dictionary"></pre>
+        </div>
+        <%= htmlWebpackPlugin .tags .bodyTags .filter( (tag) =>
```
Comment (nicholasrice) [inline sites/fast-tooling-examples/app/examples/native-element-1/index.html:16]: hmm this looks like it might error now - does this still work??

Comment (nicholasrice) [reply sites/fast-tooling-examples/app/examples/native-element-1/index.html:16]: huh... nevermind I  think it's fine. I guess spaces between property accessors is legal

Comment (janechu) [reply sites/fast-tooling-examples/app/examples/native-element-1/index.html:16]: Still works fine, I suppose it's treated like js where you can line break at will for chained methods and object properties

File: packages/web-components/fast-components/src/radio-group/fixtures/base.html (hunk 3 above)
Comment (EisenbergEffect) [reply packages/web-components/fast-components/src/radio-group/fixtures/base.html:7]: Definitely not 🤣

Comment (janechu) [reply packages/web-components/fast-components/src/radio-group/fixtures/base.html]: Filed an issue for it https://github.com/microsoft/fast/issues/3505

**Pull request merged**